### PR TITLE
✨ (Student Profile): Created Student Profile directory

### PR DESCRIPTION
**Pull request by caaaramia**
Created the Student-Profile directory for UI work to begin, and in order to allow other teams to access the feature to add their own components and code.

### CODE REVIEW

**Comment by Alexander-Muendesi** [comment]
In the event today ends with no review you might have to consider doing an internal review by yourselves otherwise your PR will go stale as Mr Arne seems to be not that active currently reviewing PRs from what I have been seeing the past week. @DanieldeNobrega  @caaaramia  @MildogMiller

**Comment by arneschreuder** [comment]
Wait, are you waiting for me? Why?

File: package.json [hunk 1]
```diff
@@ -91,7 +91,11 @@
     "eslint-plugin-cypress": "^2.10.3",
     "jest": "27.2.3",
     "jest-preset-angular": "11.1.1",
+    "ng-packagr": "^13.0.0",
```
Comment (arneschreuder) [inline package.json:94]: What are these?

**Review by arneschreuder — COMMENTED** [review]
Please do not wait for me. Review within your teams. There are a lot of people that have eyes on the PRs. It shouldn't just be me.

**Review by DanieldeNobrega — APPROVED** [review]

**Review by DanieldeNobrega — APPROVED** [review]

**Review by DanieldeNobrega — APPROVED** [review]

**Pull request merged**